src/AssetsPage/AssetsTable/AssetsTable.test.jsx

```diff
@@ -74,7 +81,31 @@ describe('<AssetsTable />', () => {
       expect(wrapper.find('tr')).toHaveLength(defaultProps.assetsList.length + 1);
     });
     it('correct number of trashcan buttons', () => {
-      expect(wrapper.find('button').filterWhere(button => button.hasClass('fa-trash'))).toHaveLength(defaultProps.assetsList.length);
+      expect(wrapper.find('tr button').filterWhere(button => button.hasClass('fa-trash'))).toHaveLength(defaultProps.assetsList.length);
+    });
+    it('correct number of images', () => {
+      const images = wrapper.find('tr img');
+      expect(images).toHaveLength(defaultProps.assetsList.length);
+    });
+    it('correct type of images', () => {
+      const rows = wrapper.find('tr').filterWhere(row => row.find('td').exists());
+
+      expect(wrapper.find('td img')).toHaveLength(defaultProps.assetsList.length);
+
+      const baseUrl = defaultProps.courseDetails.base_url;
```

Comment:
This field is set within SFE itself, but it's not set in Studio, right? Does this need to default to an empty string so that it works when embedded in Studio?